src/main/java/wordcloud/nlp/FrequencyAnalizer.java

```diff
@@ -95,7 +99,39 @@
         return sorted.subList(0, Math.min(sorted.size(), wordFrequencesToReturn));
     }
 
-    public void setStopWords(Collection<String> stopWords) {
+    public List<WordFrequency> load(final List<String> texts,FileType fileType) {
+    	if(fileType == FileType.REGULAR)
+    		return load(texts);
+    	
+        final List<WordFrequency> wordFrequencies = new ArrayList<>();
+        // generate all word counts
+        final Map<String, Integer> cloud = calculateCloud(texts, wordTokenizer, fileType);
+
+        for(String key : cloud.keySet()) {
+            if(key.length() >= minWordLength
+                    && key.length() < MAX_LENGTH) {
+                wordFrequencies.add(new WordFrequency(key, cloud.get(key)));
+            }
+        }
+        return takeTopFrequencies(wordFrequencies);
+    }
+    
+    private Map<String, Integer> calculateCloud(List<String> texts, WordTokenizer tokenizer, FileType fileType) {
+    	final Map<String, Integer> cloud = new HashMap<>();
+        for(String text : texts) {
+            final List<String> words = Lambda.filter(stopWordFilter, tokenizer.tokenize(sanitizer.sanitize(text)));
+            int sz = words.size();
```

Comment:
i'd recommend a `final size = word.size()`. With the exception of incrementers in for-loops I rarely like to abbreviate variables :)